src/main/array/ArrayAndLoop.java

```diff
@@ -0,0 +1,102 @@
+package main.array;
+
+public class ArrayAndLoop {
+        public static void main(String[] args) {
+            iphoneLength();
+            getConcat();
+            getTrim();
+            getEndWith();
+            getToUpperCase();
+            getToLowerCase();
+            getEquals();
+
+        }
+
+    private static void iphoneLength() {
+            System.out.println("\nLENGTH:");
+            String[] iphones = {"Iphone 11", "Iphone 12", "Iphone 13"};
+            int phone = 0;
+
+            for (int i = 0; i < iphones.length; i++) {
+                phone = phone + iphones[i].length();
+
+            }
+            System.out.println("The Length would be: " + phone);
+
+        }
+
+        private static void getConcat() {
+            System.out.println("\nCONCAT:");
+            String[] conTask = new String[3];
```

Comment:
I'm not sure this naming is really explains meaning of the variable.
**stringArray** would be definetely better.